.gitignore

```diff
@@ -13,9 +13,12 @@ node_modules
 /coverage
 /coverage.json
 
-/deployments/hardhat
-/deployments/external-hardhat
-/deployments/localhost
+# deployments files
+deployments/*/*
+!deployments/bellecour/
```

Comment:
I suggest to keep old config, if we tomorrow deploy on another public network, we'll forget to commit related file